addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -41,6 +41,8 @@
                         <!-- Date & Time -->
                         <div class="o_wevent_sidebar_block">
                             <h6 class="o_wevent_sidebar_title">Date &amp; Time</h6>
+                            <div itemprop="startDate" hidden="True" t-field="event.date_begin"/>
+                            <div itemprop="endDate" hidden="True" t-field="event.date_end"/>
```

Comment:
the `hidden="True"` should work (thanks to Odoo custom CSS that hides everything that has `hidden` attribute`), but I don't think attribute value is evaluated in python (so the `True` is misleading), testing it seems that we could have `hidden=""` to have the attribute without value which should also work

also it was unexpected to me that it works, I think usually we just specify "display:none" with class "d-none", in my opinion it would be more expected to just do that:

```suggestion
                            <div itemprop="startDate" class="d-none" t-field="event.date_begin"/>
                            <div itemprop="endDate" class="d-none" t-field="event.date_end"/>
```
but both seems ok to me

Reply:
thanks for the suggestion :slightly_smiling_face: